crates/net/eth-wire/src/capability.rs

```diff
@@ -234,16 +234,6 @@ pub enum SharedCapability {
         /// the message id space.
         offset: u8,
     },
-    /// Any other unknown capability.
-    UnknownCapability {
-        /// Shared capability.
-        cap: Capability,
-        /// The message ID offset for this capability.
-        ///
-        /// This represents the message ID offset for the first message of the eth capability in
-        /// the message id space.
-        offset: u8,
-    },
```

Comment:
I want to keep this because this allows us to support any protocol without having to add it as a variant

Reply:
but we would still have to build handles for it right so then we would most likely know its name, and most likely a spec exists. do we need to keep it?